### PR TITLE
Kravchenko Dmitrii, Homework 5

### PR DESCRIPTION
**Pull request by equivalence1**
Там на самом деле все ячейки отработали с подбором, но почему-то как-то криво отображаются. В сравнениях там все модели.

Перебор прям по всем я не запускал (последнее в блоке про xgboost, если я его правильно понял), это заняло бы уж слишком много времени.

Еще там забавный момент есть -- модели после подбора параметров иногда начинают терять в качестве. Мне кажется, это из-за того, что я не всегда включал то значение, которое было установлено в baseline, в перебор.

### CODE REVIEW

**Comment by ishugaepov** [comment]
Уберите лишние файлы из PR'a. (можно rebase'ом на свежий мастер основного репозитория)

**Comment by equivalence1** [comment]
@ishugaepov 
done. Забыл синхронуть мастер с апстримом

**Comment by ishugaepov** [comment]
* Tune gamma - используете `max_evals` равным 20, в результате по несколько раз обучаете модель на одних и тех же гиперпараметрах.
* Lower the learning rate and decide the optimal parameters - тут вы не так поняли задание. Нужно было уменьшить lr и руками немного покрутить параметры, например увеличить число деревьев. Можно было почитать материалы, ссылки на которые есть прямо в тетрадке.
* Не построена итоговая таблица с результатами
* Везде где выводите `all_metrics` показываете итоговый результат
* Нет HPO для логрега

0.55
